build.gradle.kts

```diff
@@ -1,5 +1,5 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import io.gitlab.arturbosch.detekt.Detekt
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     id("java-gradle-plugin")
```

Comment:
Can we use `java-gradle-plugin` instead of `id("java-gradle-plugin")`?

Reply:
yes but this does not matter ;)

Reply:
sure, I think a backtip version is nicer, but this is only formatting / style habits. OK, I don't touch it.